modules/png-extraction/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ravirahar/arch-python-gdcm
```

Comment:
This should come from a standard base image (such as Ubuntu, Python, ...). Not from your personal arch-python-gdcm base.

Reply:
Okay, I will make the changes and make pull request again. It was just that I wanted to use lightweight linux and some modules on arch, like gdcm were only on AUR. But as you say, I will use some standard image.